src/test/java/tests/bankManager/BankManagerAccountTests.java

```diff
@@ -38,8 +38,33 @@ public void homePageButtonTest() {
     @Test
     public void addCustomerButtonTest() {
         homePage.bankManagerLoginButtonClick();
-        bankManagerLoginPage.addCustomerLoginButtonClick();
+        bankManagerLoginPage.addCustomerLoginBtnClick();
         assertions.assertUrlContains("addCust");
     }
 
+    @Test
+    public void addNewCustomerTest() {
+        homePage.bankManagerLoginButtonClick();
+        bankManagerLoginPage
+                .addCustomerLoginBtnClick()
+                .firstNameFieldFilling()
+                .lastNameFieldFilling()
+                .postCodeFieldFilling()
+                .addCustomerSmallBtnClick()
+                .openAccountBtnClick()
+                .dropdownCustomerMenuClick()
+                .dropdownCurrencyMenuClick(1)
+                .processBtnClick()
+                .dropdownCustomerMenuClick()
+                .dropdownCurrencyMenuClick(2)
+                .processBtnClick()
+                .dropdownCustomerMenuClick()
+                .dropdownCurrencyMenuClick(3)
+                .processBtnClick()
+                .customersBtnClick()
```

Comment:
take out in the method. And method should take an argument pattern builder. Example: public void createCustomer(Customer customer) {}